contributing.md

```diff
@@ -0,0 +1,141 @@
+
+<!-- /code_chunk_output -->
+
+
+## Preparing the development environment
+### Fork the project
+* Fork the repo.
+* **Always create a new branch before working on your fork.**
+* You can now improve the website.
+
+> You can use a git client like [GitHub Desktop](https://desktop.github.com/) to manage your local repository.
+
+When your done submit a pull request.
+
+### Running the website
+> When you make modifications to the site it can be useful to render the code. You will thus see how the modifications look like.
+
+See [Building local enviroment](https://github.com/Paperback-iOS/website#building-local-enviroment)
+
+## Guidelines
+### Glossary
+
+ * App/application --> app
+ * Device/iPhone/iPad/phone --> iPhone or iPad
+ * iOS --> iOS or iPadOS
+
+> Be careful of capitalization:
+
+* Paperback
+* MangaDex
+
+* AltStore
+* AltServer
+
+* Safari
+* macOS
+
+### Syntax and layout
+> To create a numbered list use `1.` on all lines:
+```markdown
+1. Line 1
+1. Line 2
+```
+
+> Write `.IPA` in a code block.
+
+> Use `[Image]()` or `[GIF]()` to link to an explicative picture.
+
+### "Not up to date" warning
+> You've made a major modification to the English version of the site and you want to show that a translation is not up to date?
+> You can  mark a page as "not up to date" by adding the following code at the top of the concerned page.
+
+```html
+<notuptodate/>
+```
+It will create a translated warning saying:
+> Major changes have been made in the English version of this page. We suggest you to use this one.
+
+### Components
+You can use the following components. Try not to abuse of `warning`.
+```markdown
+::: tip Title
+ Content
+:::
+```
+```markdown
+::: warning Title
+ Content
+:::
+```
+```markdown
+:::: el-tabs
+::: el-tab-pane label="Tab1"
+Content1
+:::
+::: el-tab-pane label="Tab2"
+Content2
+:::
+::::
+```
+```html
+<el-tag type="warning">Not the Windows Store app!</el-tag>
+```
+```html
+<Download text="from here"/>
+```
+```html
+<notuptodate/>
+```
+### Potential issue
+**Note that `tabs` currently needs to begin by a `h3` and finish by an `ol` components.** 
+Otherwise there will be padding issues. See [PR #20](https://github.com/Paperback-iOS/website/pull/20)
+
+## Localization
+You want to translate the website?
+The following languages are currently available:
+|  Language  |  Translator  |    Reviewer    |
+|:----------:|:------------:|:--------------:|
+|   English  |              |                |
+|   French   |     Lemon    |  Sirus, fma16  |
+|  Italiano  |   Kinpatsu   |                |
+| Nederlands |     Joery    |       Syn      |
+
+
+In development: 
+|  Language  |  Translator  |    Reviewer    |
+|:----------:|:------------:|:--------------:|
+|  Russian   |KorewaWatchful|     Dkzver     |
+
+
+### Adding a new language
+* Copy `website/src/README.md` to `website/src/LANG/README.md` and `website/src/help` to `website/src/LANG/help`.
+* Don't forget to edit `.vuepress` folder.
```

Comment:
What should they be editing inside of the folder?

Reply:
I have try to improve this part with cfdb9b2. What do you think?

```markdown
### Adding a new language
 * Copy the following files:
   * `website/src/README.md` to `website/src/LANG/README.md`
   * `website/src/help` to `website/src/LANG/help`
   * `src/.vuepress/config/nav/en.js` to `src/.vuepress/config/nav/LANG.js`
   * `src/.vuepress/config/sidebar/en.js` to `src/.vuepress/config/sidebar/LANG.js`
 * Translate these files.
 * Edit `src/.vuepress/config.js` file the same way it was done for the other translations.

Don't forget to change links:
`/help/faq/#troubleshooting` -> `/fr/help/faq/#resolution-des-problemes-de-l-application`
```

Reply:
I suggest you still be a little more clear about what you should be editing inside of `src/.vuepress/config.js` - You state 'the same way it was done for other translations' which is a little vague. The goal of documentation is to be explicit.

The rest of the instructions seem to be very clear!

Reply:
You're right!
I tried to explain more precisely what should be done in aa7f819.
I found it difficult to make it clear, so if you have some suggestions or feedback, I will be happy to add them.